### PR TITLE
docs(classes): add comment about limitation of array properties

### PR DESCRIPTION
**Pull request by huybn5776**
#close 309

### CODE REVIEW

File: docs-site/docs/plugins-system/classes-limitations.md [hunk 1]
```diff
@@ -190,3 +190,50 @@ export class User {
...
+  @AutoMap()
+  profile!: UserProfile;
+  @AutoMap({ typeFn: () => Job }) // <- it's required for nested array property
+  jobs!: Job[];
+}
+
+mapper.createMap(User, UserVm);
+```
+
+AutoMapper will get property type through reflection, in most cases it will work without any problem, but for array property, it will only result an  `Array` type instead of `Job[]` above. It's currently an [open issue](https://github.com/microsoft/TypeScript/issues/7169) of TypeScript.
```
Comment (nartc) [inline docs-site/docs/plugins-system/classes-limitations.md:213]: suggestion: 
- Let's use "AutoMapper will attempt to get the data type of the property..."
- `in most cases` should be the start of a sentence. Like `...through reflection. In most cases, ...`

**Review by nartc — COMMENTED** [review]
Thank you. I only have 1 nitpick and 1 suggestion.

File: docs-site/docs/getting-started/introduce-to-automapper.md [hunk 2]
```diff
@@ -208,6 +208,8 @@ We did the following:
 - Removed the mapping logic in the DTOs
 - Decorated all the properties with the same name with `@AutoMap()`. For nested model like `BioDto`, we provide a `typeFn` to `@AutoMap()`
 
+You can omit `typeFn` for non-array nested model, but for array property it's may be required, for more detail please check [Classes Limitations](plugins-system/classes-limitations/#nested-array-property)
```
[the remark this answers is not in the document]
Comment (huybn5776) [reply docs-site/docs/getting-started/introduce-to-automapper.md]: ✅

File: docs-site/docs/plugins-system/classes-limitations.md (hunk 1 above)
Comment (huybn5776) [reply docs-site/docs/plugins-system/classes-limitations.md]: ✅

**Review by nartc — APPROVED** [review]
LGTM! Thank you for the PR

**Pull request merged**